scripts/build-data.js

```diff
@@ -7,9 +7,36 @@ const flatten = require("lodash/flatten");
 const flattenDeep = require("lodash/flattenDeep");
 const mapValues = require("lodash/mapValues");
 const pickBy = require("lodash/pickBy");
+const electronToChromiumVersions = require("electron-to-chromium").versions;
 const pluginFeatures = require("../data/plugin-features");
 const builtInFeatures = require("../data/built-in-features");
 
+const electronToChromiumKeys = Object.keys(electronToChromiumVersions).reverse();
+
+const chromiumToElectronMap = electronToChromiumKeys.reduce(
+  (all, electron) => {
+    all[electronToChromiumVersions[electron]] = +electron;
+    return all;
+  }
+, {});
+const chromiumToElectronVersions = Object.keys(chromiumToElectronMap);
+
+const findClosestElectronVersion = (targetVersion) => {
+  const chromiumVersionsLength = chromiumToElectronVersions.length;
+  const maxChromium = +chromiumToElectronVersions[chromiumVersionsLength - 1];
+  if (targetVersion > maxChromium) return null;
+  const closestChrome = chromiumToElectronVersions.reduce((result, version) => {
```

Comment:
@existentialism so we need to update `chromiumToElectron`. for now, it will just return `undefined` if no explicit values were found.